src/main/resources/webapp/webapp.ts

```diff
@@ -0,0 +1,9 @@
+const thymeleaf = __non_webpack_require__('/lib/thymeleaf');
+const view = resolve('../template/main.html');
+
+export function get() {
+    log.info('Hello from transpiled Typescript server-side code');
+    return {
+        body: thymeleaf.render(view, {})
+    }
+}
```

Comment:
this is a default example file automatically installed from starter, in case we want to use enonic to make a web app application. I guess we can remove it since we are going not in this scenario